### PR TITLE
Expose `dt_storage_pool_log_gc_force_hardlink_rate` to users.

### PR DESCRIPTION
**Pull request by jiaqizho**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8610
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -193,6 +193,10 @@ delta_index_cache_size = 0
     ## New in v5.4.0. This item enables or disables the elastic thread pool feature, which significantly improves CPU utilization in high concurrency scenarios of TiFlash. The default value is false. The elastic thread pool feature is experimental and not recommended for production environments.
     enable_elastic_threadpool = false
 
+    ## New in v5.4. Max valid rate of deciding a page file(log type) can do hardlink. 
+    ## Not enable by default. It is recommended to change to 0.8, if the GC time is too long.
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:197]:
```suggestion
    ## New in v5.4.0. This item specifies the maximum valid rate for triggering a hardlink by a page file(log type). 
    ## Disabled by default. If the GC time is too long, it is recommended that you change the value to 0.8 to enable the hardlink function.
```

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -193,6 +193,10 @@ delta_index_cache_size = 0
     ## New in v5.4.0. This item enables or disables the elastic thread pool feature, which significantly improves CPU utilization in high concurrency scenarios of TiFlash. The default value is false. The elastic thread pool feature is experimental and not recommended for production environments.
     enable_elastic_threadpool = false
 
+    ## New in v5.4.0. This item specifies the maximum valid rate for triggering a hardlink by a page file (log type).
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md]:
```suggestion
    ## New in v5.4.0. This item specifies the maximum valid rate for triggering a hardlink by a PageFile (log type). 
```

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -193,6 +193,10 @@ delta_index_cache_size = 0
     ## New in v5.4.0. This item enables or disables the elastic thread pool feature, which significantly improves CPU utilization in high concurrency scenarios of TiFlash. The default value is false. The elastic thread pool feature is experimental and not recommended for production environments.
     enable_elastic_threadpool = false
 
+    ## New in v5.4.0. This item specifies the maximum valid rate for triggering a hardlink by a PageFile (log type). 
+    ## Disabled by default. If the GC time is too long, it is recommended that you change the value to 0.8 to enable the hardlink function.
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:197]:
```suggestion
    ## Defaulted to 2, which disables the hardlink function. If the GC time is too long, it is recommended that you change the value to 0.8 to enable the hardlink function.
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -193,6 +193,10 @@ delta_index_cache_size = 0
     ## New in v5.4.0. This item enables or disables the elastic thread pool feature, which significantly improves CPU utilization in high concurrency scenarios of TiFlash. The default value is false. The elastic thread pool feature is experimental and not recommended for production environments.
     enable_elastic_threadpool = false
 
+    ## New in v5.4.0. This item specifies the maximum valid rate for triggering a hardlink by a PageFile (log type).
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:196]:
```suggestion
    ## New in v5.4.0. This item specifies the maximum valid rate for triggering a hardlink by a PageFile (log type). The value range is [0.1,1.0]. The smaller the value, the more frequent the hardlink operations. The greater the value, the more strict the condition to trigger a hardlink.
```

File: tiflash/tiflash-configuration.md [hunk 5]
```diff
@@ -193,6 +193,10 @@ delta_index_cache_size = 0
     ## New in v5.4.0. This item enables or disables the elastic thread pool feature, which significantly improves CPU utilization in high concurrency scenarios of TiFlash. The default value is false. The elastic thread pool feature is experimental and not recommended for production environments.
     enable_elastic_threadpool = false
 
+    ## New in v5.4.0. This item specifies the maximum valid rate for triggering a hardlink by a PageFile (log type). The value range is [0.1,1.0]. The smaller the value, the more frequent the hardlink operations. The greater the value, the more strict the condition to trigger a hardlink.
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:196]:
```suggestion
    ## New in v5.4.0. This item specifies the minimum valid rate for triggering a hardlink by a PageFile (log type). The value range is [0.1,1.0]. The smaller the value, the more frequent the hardlink operations. The greater the value, the more strict the condition to trigger a hardlink.
```

File: tiflash/tiflash-configuration.md [hunk 6]
```diff
@@ -193,6 +193,10 @@ delta_index_cache_size = 0
     ## New in v5.4.0. This item enables or disables the elastic thread pool feature, which significantly improves CPU utilization in high concurrency scenarios of TiFlash. The default value is false. The elastic thread pool feature is experimental and not recommended for production environments.
     enable_elastic_threadpool = false
 
+    ## New in v5.4.0. This item specifies the minimum valid rate for triggering a hardlink by a PageFile (log type). The value range is [0.1,1.0]. The smaller the value, the more frequent the hardlink operations. The greater the value, the more strict the condition to trigger a hardlink.
```
Comment (qiancai) [inline tiflash/tiflash-configuration.md:196]:
```suggestion
    ## New in v5.4.0. This item specifies the minimum valid rate for triggering a hardlink by a PageFile (log type). The value range is [0.1,1.0]. The smaller the value, the more frequent to trigger a hardlink. The greater the value, the more strict condition to trigger a hardlink.
```

File: tiflash/tiflash-configuration.md [hunk 7]
```diff
@@ -193,6 +193,10 @@ delta_index_cache_size = 0
...
+    ## Defaulted to 2, which disables the hardlink function. If the GC time is too long, it is recommended that you change the value to 0.8 to enable the hardlink function.
```
Comment (qiancai) [inline tiflash/tiflash-configuration.md:197]:
```suggestion
    ## The default value is 2, which disables the hardlink function. If the GC time is too long, it is recommended that you change the value to 0.8 to enable the hardlink function.
```

Comment (qiancai) [reply tiflash/tiflash-configuration.md:197]: The word "default" is rarely used as a verb in user documentation

**Pull request closed without merging**